README.md

```diff
@@ -19,3 +19,10 @@ Make sure you have golang installed
 deploy`
 7. To verify deployed project:
 `curl https://go-whiteboard.appspot.com/helloWorld`
+
+## Writing and Compiling .proto files into .go
```

Comment:
In the future we'll probably need to add a build script that will "do the right thing" for regenerating all the proto files in the project so that the person, changing protos, don't need to remember how to do it.